drivers/storage/portworx/portworx.go

```diff
@@ -772,6 +788,8 @@ func setCSISpecDefaults(toUpdate *corev1.StorageCluster) {
 			Enabled:                   pxutil.FeatureCSI.IsEnabled(toUpdate.Spec.FeatureGates),
```

Comment:
Still looking at the feature gate here?

Reply:
Yeah- before, we were enabling CSI if one of the two were set:
1. feature flag for CSI is enabled
2. CSI Spec has enabled flag as true

So, it was a logical `or` with the above flags. If one was set, enable CSI. To keep backwards compatibility, we need to read the feature flag, set CSI spec as enabled, and delete the feature flag.

Reply:
true, will remove this.